building/devices/qemu.rst

```diff
@@ -115,6 +115,108 @@ Now GDB is connected to the remote application. You may use GDB normally.
     (gdb) b main
     (gdb) c
 
+GDB - Secure world
+******************
+TEE core debugging
+==================
+To debug TEE core running QEMU with GDB, you don't have to enable any special
+flags as such, but it's easier to debug if you have optimization disabled. Other
+than that you will have four consoles that you are working with.
+
+    - Qemu console
+    - NW UART console
+    - SW UART console
+    - GDB console
+
+All of them but the GDB console are consoles you normally will see/use when
+running OP-TEE/xtest using QEMU. The first thing is to start QEMU, i.e.,
+
+.. code-block:: bash
+
+    $ cd <qemu-v7-project>/build
+    # make run-only also works if you don't want to rebuild things
+    $ make run
+
+Next launch another console for GDB and do this
+
+.. code-block:: bash
+
+    $ cd <qemu-v7-project>/toolchains/aarch32/bin
+    $ ./arm-linux-gnueabihf-gdb -q
+
+In the GDB console connect to the QEMU GDB server, like this (the output is
+include to show what you normally will see).
```

Comment:
included

Reply:
Thanks, fixed while force pushing and adding your tag.